frontend/packages/helm-plugin/src/components/details-page/history/HelmReleaseHistory.tsx

```diff
@@ -28,20 +28,26 @@ const HelmReleaseHistory: React.FC<HelmReleaseHistoryProps> = ({ match, obj }) =
 
   React.useEffect(() => {
     let destroyed = false;
-    coFetchJSON(`/api/helm/release/history?ns=${namespace}&name=${helmReleaseName}`)
-      .then((items) => {
-        if (!destroyed) {
-          setLoadError(null);
-          setRevisionsLoaded(true);
-          setRevisions(items);
-        }
-      })
-      .catch((err) => {
-        if (!destroyed) {
-          setRevisionsLoaded(true);
-          setLoadError(err.message || t('helm-plugin~Unable to load Helm Release history'));
-        }
-      });
+    if (parseInt(memoizedObj.metadata.labels.version, 10) > 1) {
```

Comment:
We should still show the history list. Just the kebab action should not be shown if it only has one revision.